src/utils/cacheble.ts

```diff
@@ -0,0 +1,20 @@
+interface Cache {
+    [key: string]: any;
+}
```

Comment:
во многих случаях `any` можно заменить на `unknown`

здесь еще можно воспользоваться утилитным типом TS:
```suggestion
type Cache = Record<string, unknown>;
```